apache/docker-entrypoint.sh

```diff
@@ -13,22 +13,6 @@ EOT
     fi
 fi
 
-if [ ! -z "${HIDE_PHP_VERSION}" ]; then
-    echo "PHP version is now hidden."
-    echo -e 'expose_php = Off\n' > $PHP_INI_DIR/conf.d/phpmyadmin-hide-php-version.ini
-fi
-
-UPLOAD_LIMIT_INI_FILE="$PHP_INI_DIR/conf.d/phpmyadmin-upload-limit.ini"
-if [ ! -z "${UPLOAD_LIMIT}" ]; then
-    echo "Adding the custom upload limit."
-    echo -e "upload_max_filesize = $UPLOAD_LIMIT\npost_max_size = $UPLOAD_LIMIT\n" > $UPLOAD_LIMIT_INI_FILE
-fi
-
-if [ ! -z "${MAX_EXECUTION_TIME}" ]; then
-    echo "Adjusting the maximum execution time."
-    sed -i "s/^max_execution_time = .*$/max_execution_time = ${MAX_EXECUTION_TIME}/g" $PHP_INI_DIR/conf.d/phpmyadmin-misc.ini
-fi
-
```

Comment:
Why are the blocks removed ?

Default values are defined at image build time, okay
But what about ENV variables using for compose or run will not work, right ?

Reply:
PHP is now parsing ENVs directly: https://www.php.net/manual/en/configuration.file.php#example-36

Reply:
Okay, could you please add an unit test to make this be a safety and self explanatory for any dev not knowing that ?

Reply:
But will that work because the ENVs are not `PHP_` named :thinking: ?

Reply:
Be sure to test the ENV too ;)